amplipi/tests.py

```diff
@@ -4,21 +4,23 @@
 
 from time import sleep
 from typing import Optional, Sequence
-import sys
-import subprocess
-import signal
 import argparse
 import requests
+import shlex
+import signal
+import subprocess
+import sys
```

Comment:
:sunglasses:

Reply:
These 4 imports make me feel like I should just be using Bash